--- .zshrc	2019-05-04 17:56:43.226981954 +0200
+++ .zshrc-new	2019-05-04 17:57:41.235971968 +0200
@@ -8,7 +8,7 @@
 # load a random theme each time oh-my-zsh is loaded, in which case,
 # to know which specific one was loaded, run: echo $RANDOM_THEME
 # See https://github.com/robbyrussell/oh-my-zsh/wiki/Themes
-ZSH_THEME="robbyrussell"
+ZSH_THEME="agnoster"

 # Set list of themes to pick from when loading at random
 # Setting this variable when ZSH_THEME=random will cause zsh to load
@@ -62,7 +62,14 @@
 # Custom plugins may be added to ~/.oh-my-zsh/custom/plugins/
 # Example format: plugins=(rails git textmate ruby lighthouse)
 # Add wisely, as too many plugins slow down shell startup.
-plugins=(git)
+plugins=(
+  git
+  composer
+  docker
+  docker-compose
+  symfony
+  zsh-autosuggestions
+)

 source $ZSH/oh-my-zsh.sh

@@ -91,3 +98,7 @@
 # Example aliases
 # alias zshconfig="mate ~/.zshrc"
 # alias ohmyzsh="mate ~/.oh-my-zsh"
+
+DEFAULT_USER=$USER
+
+alias zshconfig="nano ~/.zshrc"